tests/bluetooth/tester/qemu.conf

```diff
@@ -18,5 +18,17 @@ CONFIG_BT_DEBUG_ATT=y
 CONFIG_BT_DEBUG_GATT=y
 CONFIG_BT_L2CAP_DYNAMIC_CHANNEL=y
 CONFIG_BT_PRIVACY=y
+
+CONFIG_BT_MESH=y
+CONFIG_BT_MESH_PROV=y
+CONFIG_BT_MESH_RELAY=n
+CONFIG_BT_MESH_PB_ADV=y
+CONFIG_BT_MESH_PB_GATT=n
+CONFIG_BT_MESH_GATT_PROXY=n
+CONFIG_BT_MESH_SUBNET_COUNT=1
+CONFIG_BT_MESH_APP_KEY_COUNT=1
+CONFIG_BT_MESH_MODEL_GROUP_COUNT=2
+CONFIG_BT_MESH_LABEL_COUNT=3
```

Comment:
Same thing here. Looks like a super limited configuration. Especially for qemu there should be no reason to try to stay slim since there's plenty of memory available.

Reply:
You have right. I changed this in the recent version of this patch.